
From: janitor@sternwelten.at

Change units of ICN_BOOT_TIMEOUT1 to msecs instead of jiffies for
msleep_interruptible() in icn.c.  Also, remove unused constant
ICN_CHANLOCK_DELAY.

Signed-off-by: Nishanth Aravamudan <nacc@us.ibm.com>
Signed-off-by: Maximilian Attems <janitor@sternwelten.at>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/isdn/icn/icn.h |    3 +--
 1 files changed, 1 insertion(+), 2 deletions(-)

diff -puN drivers/isdn/icn/icn.h~janitor-isdn-icn-change-units-of-icn_boot_timeout1 drivers/isdn/icn/icn.h
--- 25/drivers/isdn/icn/icn.h~janitor-isdn-icn-change-units-of-icn_boot_timeout1	Thu Sep 23 17:06:05 2004
+++ 25-akpm/drivers/isdn/icn/icn.h	Thu Sep 23 17:06:05 2004
@@ -70,8 +70,7 @@ typedef struct icn_cdef {
 #define ICN_FLAGS_RUNNING  4    /* Cards driver activated                  */
 #define ICN_FLAGS_RBTIMER  8    /* cyclic scheduling of B-Channel-poll     */
 
-#define ICN_BOOT_TIMEOUT1  (HZ) /* Delay for Boot-download (jiffies)       */
-#define ICN_CHANLOCK_DELAY (HZ/10)	/* Delay for Channel-mapping (jiffies)     */
+#define ICN_BOOT_TIMEOUT1  1000 /* Delay for Boot-download (msecs)         */
 
 #define ICN_TIMER_BCREAD (HZ/100)	/* B-Channel poll-cycle                    */
 #define ICN_TIMER_DCREAD (HZ/2) /* D-Channel poll-cycle                    */
_
